common/cmake/devices.cmake

```diff
@@ -242,6 +242,8 @@ function(DEFINE_DEVICE_TYPE)
   #   DEVICE_TYPE <device_type>
   #   ARCH <arch>
   #   ARCH_XML <arch.xml>
+  #   [SCRIPT_OUTPUT_NAME]
+  #   [SCRIPTS]
```

Comment:
This needs some documentation updating right?

Reply:
Added some